content/blog/part-7-how-collaborative-teams-of-agents-unlock-new-intelligence.md

```diff
@@ -0,0 +1,191 @@
+---
+title: "Part 7: How Collaborative Teams of Agents Unlock New Intelligence"
```

Comment:
I think the title should be sentence case. 
You may want to wait until Dale review the post for the exact formatting of the title;

My suggestion:
Part 7: How Collaborative Teams of Agents unlock new intelligence